benchmark_runner/common/oc/oc.py

```diff
@@ -715,6 +717,20 @@ def collect_events(self):
         self.run(f"{self.__cli} get events -A > '{output_filename}' ")
         return output_filename
 
+    def get_kube_api_server(self):
+        try:
+            with open(self.__kubeconfig_path, "r") as f:
+                config = yaml.safe_load(f)
+
+            # Extract the API server URL from the clusters section
+            return config["clusters"][0]["cluster"]["server"]
```

Comment:
This is probably going to be OK in practice since you have control over the kubeconfig file and there isn't likely to be more than one cluster in the file, but you may want to consider using name of the cluster for lookup.  I'm going to approve it, but it may be a problem down the road at some point.